.github/actions/composite/setupNode/action.yml

```diff
@@ -29,36 +29,27 @@ runs:
       shell: bash
       run: |
         set -e
-        if [[ `git diff main --name-only | grep \.patch` != null ]]; then 
+        git fetch origin main --no-tags --depth=1
```

Comment:
We did not have to fetch here before. Why did we add this?

Reply:
The problem can be seen in [this](https://github.com/Expensify/App/actions/runs/5983768100/job/16234272712?pr=25988) job. In this job, the git diff fails with the following

fatal: ambiguous argument 'main': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

Due to missing main branch in gh checkout. And it somehow sets CHANGES_IN_PATCH_FILES to true, which is running npx patch-package even if there are no changes in patch files.